source/future.tex

```diff
@@ -1271,7 +1271,7 @@
 To enable old function adaptors to manipulate function objects
 that take one or two arguments,
 many of the function objects in this standard
-correspondingly provide typedefs
+correspondingly provide type aliases
```

Comment:
Thanks - I think I may want to apply this recommendation in several places that I did not touch, as I did not have a better way to update the existing wording talking about typedefs.  I'll take another pass tonight.